plugins/LadspaEffect/CMakeLists.txt

```diff
@@ -4,22 +4,22 @@ BUILD_PLUGIN(ladspaeffect LadspaEffect.cpp LadspaControls.cpp LadspaControlDialo
 
 SET(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/ladspa")
 
-IF(WANT_CAPS)
+IF(LMMS_HAVE_CAPS)
 ADD_SUBDIRECTORY(caps)
-ENDIF(WANT_CAPS)
+ENDIF()
 
-IF(WANT_TAP)
+IF(LMMS_HAVE_CAPS)
 ADD_SUBDIRECTORY(tap)
-ENDIF(WANT_TAP)
+ENDIF()
 
-IF(WANT_SWH)
+IF(LMMS_HAVE_SWH)
 ADD_SUBDIRECTORY(swh)
-ENDIF(WANT_SWH)
+ENDIF()
 
-IF(WANT_CMT)
+IF(LMMS_HAVE_SWH)
```

Comment:
```suggestion
IF(LMMS_HAVE_CMT)
```

Reply:
Whoops, messed up the order. Thanks for catching.